src/graph/graph_hub.c

```diff
@@ -183,6 +185,17 @@ uint DeleteNode
 		_DeleteNodeFromIndices(gc, n);
 	}
 
+	if(deleted_labels_count) {
+		uint label_count;
+		NODE_GET_LABELS(gc->g, n, label_count);
+		*deleted_labels_count = label_count;
+	}
+
+	if(deleted_properties_count) {
+		const AttributeSet attributes = GraphEntity_GetAttributes(n);
+		*deleted_properties_count = attributes ? attributes->attr_count : 0;
```

Comment:
talk with @swilly22

Reply:
Please skip this statistic, I don't see a really value in reporting the number of attributes removed, (also Neo doesn't report that) if needed adjust TCK accordingly.